test/production/next/jest/index.test.ts

```diff
@@ -8,13 +8,23 @@ describe('next/jest', () => {
   beforeAll(async () => {
     next = await createNext({
       files: {
+        'public/vercel.svg':
+          '<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"/>',
+        'components/image.js': `
+          import Image from "next/image"
+          import img from "../public/vercel.svg"
+          export function Img() {
+            return <Image src={img} alt="logo" width={24} height={24} />
```

Comment:
Does the mock work correctly when not manually providing the `width` and `height` properties like we [document here](https://nextjs.org/docs/basic-features/image-optimization#local-images), can we add a test for this case?

Reply:
Good catch. I had a look and I did need to add an extra case in the code similar to:

https://github.com/vercel/next.js/blob/bb3ae263550788843c4e4e2b42ef1befdf224e6f/packages/next/client/image.tsx#L857